pandas/tests/io/excel/test_readers.py

```diff
@@ -1759,3 +1759,53 @@ def test_corrupt_files_closed(self, engine, tmp_excel):
                 pd.ExcelFile(tmp_excel, engine=engine)
             except errors:
                 pass
+
+    def test_mi_header_skiprows1(self, engine, read_ext):
+        if engine is None and read_ext == ".xlsx":
```

Comment:
Can you add references to the issue as a comment in the first line of each test:

    # GH#58898